sourced/ml/extractors/children.py

```diff
@@ -0,0 +1,126 @@
+import numpy
+import re
+from collections import defaultdict
+
+from sourced.ml.algorithms import UastNodes2Bag
+from sourced.ml.extractors import BagsExtractor, register_extractor, get_names_from_kwargs,\
+    filter_kwargs
+
+
+@register_extractor
+class ChildrenBagExtractor(BagsExtractor):
```

Comment:
Could you please add a comment which says the sequence in which the methods are invoked.